### PR TITLE
Add method signature of NilClass

### PR DESCRIPTION
**Pull request by kunitoo**
- `to_i`

### CODE REVIEW

File: stdlib/builtin/nil_class.rbs [hunk 1]
```diff
@@ -32,6 +32,13 @@ class NilClass < Object
   # ```
   def to_h: () -> ::Hash[untyped, untyped]
 
+  # Always returns zero.
+  # 
+  # ```ruby
+  # nil.to_i   #=> 0.0
```
Comment (pocke) [inline stdlib/builtin/nil_class.rbs]: nil.to_i returns `0`, not `0.0`. Could you update this comment?

File: stdlib/builtin/nil_class.rbs [hunk 2]
```diff
@@ -32,6 +32,13 @@ class NilClass < Object
...
+  # ```
+  def to_i: () -> Integer
```
Comment (hanachin) [inline stdlib/builtin/nil_class.rbs]: `nil.to_i` always returns 0, IMO literal type `0` is better than Integer

**Comment by soutaro** [comment]
@kunitoo The comments above would make sense. Could you update?

File: stdlib/builtin/nil_class.rbs (hunk 1 above)
Comment (kunitoo) [reply stdlib/builtin/nil_class.rbs]: Thank you for the review. Fixed it. https://github.com/ruby/ruby-signature/pull/66/commits/cdfe4e5709b3d58ec7c0905eec18cc55bf6c44ee

File: stdlib/builtin/nil_class.rbs (hunk 2 above)
Comment (kunitoo) [reply stdlib/builtin/nil_class.rbs]: Thank you for the review. I didn't know there was a literal type.
Fixed https://github.com/ruby/ruby-signature/pull/66/commits/cdfe4e5709b3d58ec7c0905eec18cc55bf6c44ee

**Comment by kunitoo** [comment]
@soutaro I updated to reflect comments.

**Pull request merged**